test/addons-napi/test_typedarray/test.js

```diff
@@ -55,3 +55,20 @@ arrayTypes.forEach((currentType) => {
   assert.notStrictEqual(theArray, template);
   assert.strictEqual(theArray.buffer, buffer);
 });
+
+arrayTypes.forEach((currentType) => {
+  const template = Reflect.construct(currentType, buffer);
+  assert.throws(() => {
+    test_typedarray.CreateTypedArray(template, buffer, 0, 129);
+  }, RangeError);
```

Comment:
Instead of just checking for `RangeError`, can you also validate the error message.

Reply:
Done